samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/QuestionPoolMessages_es.properties

```diff
@@ -178,6 +182,7 @@ t_removePool=Eliminar bater\u00eda
 t_copyQuestion=Copiar pregunta
 t_moveQuestion=Mover pregunta
 t_exportQuestion=Exportar pregunta
+t_showSolution=Ver soluci\u00f3n
```

Comment:
Mostrar soluci\u00f3n.

Reply:
Added all messages on _ca and _eu.
Please review both languages. Thanks! :)